opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -317,6 +317,7 @@ def test_port_nested_domain(self):
                    "interface-name": 'qpi',
                    "mac": 'aa:bb:cc:00:11:22',
                    "access-interface-vlan": 4094,
```

Comment:
is it ok to just assume its true when it sees the access-interface-vlan value being present?

Reply:
Unfortunately not - there are two different behaviors that we need with the access-interface-vlan: one for nested virtualization (which needs this flag present and set to True), and the other with trunks (which doesn't need this flag present, or present and set to False). The opflex-agent does different flow-mods in the two cases.

Reply:
Got it.